opyn/opyn/contract.py

```diff
@@ -47,7 +46,11 @@ def abi_file_path(self):
         return Path(__file__).resolve().parent.parent / self.abi_location
 
     def __init__(self, config: ContractConfig):
-        if config.chain_id not in Chains:
+
+        # Can't be imported on top due to a circular dependency
+        from opyn.config import OpynSDKConfig
```

Comment:
wdyt about this? We have a circular dependency between venue/contract.py <-> venue/config.py on both Ribbon and Opyn.

We can prevent the circular dependency by importing here instead of on top of the file or we have to complicate a bit the configuration by creating an additional file to only define the supported_chains list and then import it both here and in the config

Reply:
It's fine here for now for me